app/services/curator/filestreams/iiif_manifest_prewarm_service.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Curator
+  class Filestreams::IIIFManifestPrewarmService < Services::Base
+    include Curator::Services::RemoteService
+    include Curator::Filestreams::IIIFServerHealthCheck
+
+    self.base_url = Curator.config.iiif_server_url
+    self.default_headers = { content_type: 'application/json' }
+    self.timeout_options = Curator.config.default_remote_service_timeout_opts.merge(read: 300)
+
+    attr_reader :ark_id
+
+    def initialize(ark_id)
+      @ark_id = ark_id
+    end
+
+    def call
+      begin
+        service_response = self.class.with_client do |client|
+          call_iiif_manifest_endpoint(client)
+        end
+
+        return service_response
+      rescue HTTP::Error => e
+        base_message = 'HTTP Error Occurred Calling IIIF Server'
+        json_reason = { 'reason' => e.message }.as_json
+        Rails.logger.error base_message
+        Rails.logger.error "Reason: #{e.message}"
+        raise Curator::Exceptions::RemoteServiceError.new(base_message, json_reason, 500)
+      rescue Curator::Exceptions::RemoteServiceError => e
+        Rails.logger.error 'Error Occurred Invalidating IIIF Server Cache'
+        Rails.logger.error "Reason: #{e.message}"
+        Rails.logger.error "Response code: #{e.code}"
+        Rails.logger.error "Response: #{e.json_response}"
+        raise
+      end
+      nil
+    end
+
+    protected
+
+    def call_iiif_manifest_endpoint(client)
```

Comment:
Rename method to `call_iiif_info_endpoint(client)` and replace "manifest" with "info.json" in output messages